src/internal/common/ptr/pointer.go

```diff
@@ -41,7 +45,7 @@ func OrNow(t *time.Time) time.Time {
 		return time.Now().UTC()
 	}
```

Comment:
at this point it sort of feels like we should be setting t and feeding it into `dttm.Now` as well. I'd like to be able to treat the call to the dttm package as a black box and that means I don't want to assume it's not doing additional formatting or such
```suggestion
		t = &time.Time{}
	}
```

Reply:
Well, it's permanently breaking Incrementals.  Nonetheless, doomsaying reduced.